.project

```diff
@@ -0,0 +1,3 @@
+{
+	'srcDirectory' : 'src'
```

Comment:
What does this file belong to and why is it introduced now?

Reply:
The commit message could have elaborated on the purpose of this.

Reply:
Sorry for confusion. This file explains the default directory of the package directory to the newest Metacello version so you do not need to specify the path manually in the load script.